src/dropdown.ts

```diff
@@ -2585,72 +2601,68 @@ export default class GlideCoreDropdown
   }
 
   #show() {
-    if (this.#optionElements.length > 0) {
```

Comment:
Only change is I removed this conditional because now we have something to show ("No options available") when there are no options.